client/src/testFixtures/java/org/apache/cassandra/sidecar/client/SidecarClientTest.java

```diff
@@ -1811,6 +1816,90 @@ void testNodeUpdateNative() throws Exception
         });
     }
 
+    @Test
+    public void testLiveMigrationListInstanceFiles() throws ExecutionException, InterruptedException, JsonProcessingException
```

Comment:
nit: please omit `public`, since it is unnecessary for the test.

Reply:
done